simulations/updateGraphRender.html

```diff
@@ -0,0 +1,2 @@
+<div class="graph" style="height: 100%"></div>
```

Comment:
I believe all contents under `simulations` are related to testing. Right?

Reply:
Yes. `simulations/updateGraphRender.*` is used by `tests/updateGraphRender.ts`. Simulations like that will need build steps so I was unsure how to immediately file them and put them in a directory.